packages/components/package.json

```diff
@@ -20,9 +20,10 @@
     "url": "git+https://github.com/chanzuckerberg/edu-design-system.git"
   },
   "scripts": {
-    "build": "npm run build:clean && npm run build:js && npm run build:styles && npm run build:types && rm -rf dist/playground",
+    "build": "npm run build:clean && npm run build:js && npm run build:styles && npm run build:types && npm run build:finish",
     "build:clean": "rm -rf dist/",
-    "build:js": "babel src -d dist --extensions '.js,.jsx,.ts,.tsx' --ignore src/**/*.spec.ts,src/**/*.spec.tsx --ignore src/**/*.stories.ts,src/**/*.stories.tsx",
```

Comment:
Adding single quotes would correctly ignore these for `babel`:
```
--ignore 'src/**/*.spec.ts','src/**/*.spec.tsx','src/**/*.stories.ts','src/**/*.stories.tsx'
```

but:
- `tsc` was still generating `.d.ts` files 😬 
- postcss was still including the `.stories.module.css` files 😬 

Instead of adding this custom ignore logic to every step (and partly because the babel glob matching is already bad), it felt cleaner to just remove everything with `.spec.` or `.stories.` at the end. A side benefit is we don't need to worry about where `.stories.module.css` files are included (and can move the button ones in `.storybook/Button` back)

Reply:
I like it 👍